lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -511,6 +513,16 @@ instance FromText PoolId where
         where
             textDecodingError = Left . TextDecodingError . show
 
+-- | Coins are stored as Lovelace (reminder: 1 Lovelace = 1e-6 ADA)
+newtype Stake = Stake { getStake :: Quantity "lovelace" Word64 }
```

Comment:
What's the need for the newtype wrapper here :thinking: ?

Reply:
yes, you are right. Resign from Stake